gradle/installViaTravis.sh

```diff
@@ -15,7 +15,7 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
 
   source ~/.nvm/nvm.sh
   nvm install $NODE_JS_VERSION
-  yarn --frozen-lockfile
+  yarn --ignore-optional --frozen-lockfile
```

Comment:
I don't like this. Adding a version of `fsevents` to `resolutions` seems to be overriding the `optionalDependencies` entry. 

I'm going to rework this. We don't want to have to explain to non-Mac users that they have to add `--ignore-optional` when running `yarn`.